README.md

```diff
@@ -1 +1,205 @@
-# Math-MAgicians (Microverse)
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+#Math-maigicians
+
+Create with the aim of mentioning one of the thematics of events that I like the most that is that of the video games and using all my knowledge in the right from the first module of the program microverse to give it form and functionality. 
+
+Will have future changes in the next days! 
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3 id="title"><b>Math-magicians</b></h3>
+
+</div>
+
+# 📖 Web page for Gamescom event! <a name="about-project"></a>
+
+> This is the first project of the react series im cirruntly in the aim is to use the elements of the react framework to develop a simple calculator
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- JS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+1 Add HTML <br>
+3 Add JS<br> 
+4 add linters<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p> Checkout my Awesome book   target="_blank" rel="noopener noreferrer"> here!</a> </p>  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> This is the first step in the creation of my project so there is not much yet, but it will grow!
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1 Web browser <br>
+2 Code editor   <br>
+3 git-github<br>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use these commands: 
+
+```sh
+  cd my-folder
+  git clone https://github.com/stino-x/Math-MAgicians--Microverse
+```
+--->
+
+### Install
+
+Install this project with:
+
+
+Run command:
+
+```sh
+  cd my-project
+  npm install
+  mpn create-my app react
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
```

Comment:
- [ ] Kindly update this section to be `npm start` as this is the way to start a react project 👍

Reply:
Good Job working some requirements but it would be nice to take seriously what the previous code reviewer requested you to change. A read me file should be professional at all times because its the first thing a different user that's looking at your app with go through. kindly make this changer to avoid wasting your review chances